lib/instrumentation.ex

```diff
@@ -111,6 +126,16 @@ defmodule OpentelemetryAbsinthe.Instrumentation do
     :ok
   end
 
+  defp get_operation_type(nil), do: nil
+  defp get_operation_type(document) do
```

Comment:
I can thing of a lot of ways to break this implementation(whitespace, complex queries, first letter uppercase, other exotic LangaugeAdapters), if we move this to `handle_operation_stop` we can instead make this more robust by doing 
```elixir
data.blueprint |> Absinthe.Blueprint.current_operation() |> Kernel.||(%{}) |> Map.get(:type)
```